spec/migrations/240814115647-disable_reverse_proxy_by_default_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'spec_helper'
+
+migration '240814115647-disable-reverse-proxy-by-default' do
+  scenarios %w[foreman-proxy-content] do
+    context 'reset reverse_proxy to default value to disable by default' do
+      let(:answers) do
+        {
+          'foreman_proxy_content' => {
+            'reverse_proxy' => true,
+          },
+        }
+      end
+
+      it 'changes the default to the puppet module' do
```

Comment:
`it` describes the expected behavior. How about:
```suggestion
      it 'removes the stored answer' do
```